### PR TITLE
add page for microsoft build event

### PR DESCRIPTION
**Pull request by shughes26**
## Description
New page for Microsoft Build build from kubecon template with some tweaks. [Brief here.](https://github.com/pulumi/pulumi-hugo/issues/2802)

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.
- [x] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by lindsaymarolich** [comment]
Looks good! Could we please put the date on a second line in the header so it looks like this:

Microsoft Build
May 23-25, 2023

@SaraDPH fyi on preview

File: themes/default/content/microsoft-build/_index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+meta_image: /images/microsoft-build/meta.png
+type: page
+layout: microsoft-build
+
+links:
+    items:
+        - heading: RSVP to Seattle Meetup
+          description: Join us at our re-launch of the Seattle Pulumi User Group to meet our CEO, Joe Duffy, and members of our product and engineering teams. 5PM on May 23rd at Rhein Haus Seattle.
+          action: Reserve your spot
+          link: https://www.meetup.com/pulumi-seattle/events/293452944/
```
Comment (SaraDPH) [inline themes/default/content/microsoft-build/_index.md:13]:
```suggestion
          link: https://info.pulumi.com/en/pug-meetup/happy-hour
```

File: themes/default/content/microsoft-build/_index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+title: Microsoft Build - May 23–25, 2023
+meta_desc: Join us to find out how Pulumi’s Infrastructure as Code reduces cloud complexity, accelerates development velocity and helps companies scale and innovate.
+meta_image: /images/microsoft-build/meta.png
+type: page
+layout: microsoft-build
+
+links:
+    items:
+        - heading: RSVP to Seattle Meetup
+          description: Join us at our re-launch of the Seattle Pulumi User Group to meet our CEO, Joe Duffy, and members of our product and engineering teams. 5PM on May 23rd at Rhein Haus Seattle.
```
Comment (SaraDPH) [inline themes/default/content/microsoft-build/_index.md:11]:
```suggestion
          description: No speakers, no agenda. Just a casual evening to network and have fun! Meet fellow engineers and CEO Joe Duffy and CTO Luke Lohan. Food and drinks are on us. May 23, 5:00 Pm PDT at Rhein Haus Seattle.
```

**Review by SaraDPH — APPROVED** [review]
I changed the happy hour section text and link

File: themes/default/content/microsoft-build/_index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Microsoft Build <br/> May 23–25, 2023
+meta_desc: Join us to find out how Pulumi’s Infrastructure as Code reduces cloud complexity, accelerates development velocity and helps companies scale and innovate.
+meta_image: /images/microsoft-build/meta.png
+type: page
+layout: microsoft-build
+
+links:
+    items:
+        - heading: RSVP to Seattle Meetup
```
Comment (SaraDPH) [inline themes/default/content/microsoft-build/_index.md:10]:
```suggestion
        - heading: Join Us for Happy Hour
```

**Review by SaraDPH — COMMENTED** [review]
Title changed as well

**Comment by shughes26** [comment]
> I changed the happy hour section text and link

@SaraDPH looks good! I got those changes committed so a new preview will come up soon

**Pull request merged**